lte/gateway/configs/subscriberdb.yml

```diff
@@ -17,10 +17,10 @@
 print_grpc_payload: False
 
 # Size of subscriber pages returned from cloud
-subscriber_page_size: 3000
+subscriber_page_size: 5000
 
 # Interval in seconds between gateway and cloud sync
-subscriberdb_sync_interval: 180
+subscriberdb_sync_interval: 300  # 5m
```

Comment:
What is the plan to move to delta updates?

Reply:
Remaining within the level triggered semantics, we can still compress how the levels are communicated and when the data of a particular level  is fetched. E.g., AGW can make the RPC call stating what level (e.g., version number) it is at for different buckets of subscribers, then Orc8r can stream down the buckets that have a higher version number. If we want to protect against data corruption, rather than version number, we can also use hash. I thought some level of bucketization (if not Merkle tree based approach) was on the plate.

Reply:
We are planning to support a mild form of delta updates, where the delta is computed as the set-difference between the gateway's view of the subs vs. the Orc8r's view of the subs. This will effectively be "level-triggered deltas"

However, this will not be in v1.6, as it's part of Elaine's intern project and won't be finished in time (as planned)